developer/swag-shop/introduction.md

```diff
@@ -0,0 +1,25 @@
+# Building & Launching a Swag Shop on Reaction
+
+---
+
+ **⚠ NOTICE:** This tutorial aggregates a couple of Reaction blog posts into one place that should serve our community as an accessible show case on how to build a fully customized shop on top of Reaction. The orginal blog posts can be found here:
+- <https://blog.reactioncommerce.com/building-and-launching-a-store-on-reaction/>
+- <https://blog.reactioncommerce.com/building-launching-a-swag-shop-part-2/>
+- <https://blog.reactioncommerce.com/building-a-swag-shop-lp2/>
+- <https://blog.reactioncommerce.com/building-a-swag-shop-category-tiles/>
+
+---
+
+## Introduction
+As an engineer (or manager for that matter), we've all been there: surfing the net for a shop system that actually meets our demanding needs for our next shop project. This, at least, was how I was drawn to Reaction Commerce at some distant point in the past. And, as it seems, so were you, which is why you’re currently reading this tutorial. Congratulations, you've already made the first step in the right direction.
+
+Lately, the community team has been thinking a lot about how to provide guidance to our users when it comes to the task of choosing the right ecommerce software for their next project. One of our most requested topics on our Gitter chat is a request for real-life examples on how to customize a Reaction store according to a client's specification. It didn’t take long for us to follow up on this idea into something that serves the community.
+
+My intention with this tutorial is to take you with me on our journey. From concept and design to implementation and deployment, we’ll show you how to build a shop from start to finish, from backend to frontend, finally ending up with an operational shop!
+
+## Table of Contents
+
+- [Collecting the shop requirements](/developer/swag-shop/swag-shop-collecting-requirements.md)
```

Comment:
Add Part 1, Part 2, Part 3, Part 4 here like:

- [Part 1: Collecting the shop requirements](...)

Reply:
done